dnsmasq.conf.ipv6

```diff
@@ -4,9 +4,6 @@ enable-tftp
 tftp-root=/shared/tftpboot
 log-dhcp
 
-enable-ra
-ra-param=PROVISIONING_INTERFACE,10
-
```

Comment:
Sorry, looks like we need this, I've tried running dev-scripts without it and all 3 masters downloaded iPXE but then iPXE failed.